packages/cli/src/commands/releasecelo/locked-gold.ts

```diff
@@ -43,7 +43,7 @@ export default class LockedGold extends ReleaseGoldBaseCommand {
     const releaseOwner = await this.releaseGoldWrapper.getReleaseOwner()
     const lockedGold = await kit.contracts.getLockedGold()
     kit.defaultAccount = isRevoked ? releaseOwner : beneficiary
-
+    console.log(`Using account ${kit.defaultAccount}`)
```

Comment:
why this here?

Reply:
since the account used is dependent on if the contract is revoked or not i thought showing the account used could be useful to users